clients/fossid-webapp/src/main/kotlin/model/result/LicenseCategory.kt

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
> Please name the added properties in the body text

Hint for next time: Using backticks instead of single quotes would give you nice syntax highlighting in GitHub.